packages/client/src/models/mentor.js

```diff
@@ -20,39 +29,110 @@ const MentorConverter = {
         return new Mentor(
             data.name,
             data.email,
+            data.phone,
+            data.pronouns,
+            data.college,
+            data.avatar,
+            data.bio,
+            data.major,
             data.timezone,
+            // what is this for?
```

Comment:
Does the schema have both "bio" and "about"?

Reply:
yeah